parser-core/src/main/parse/StructureParser.scala

```diff
@@ -159,11 +159,14 @@ object StructureParser {
           .filter(_.tpe == TokenType.Literal)
           .map(_.value)
           .collect {
-            case s: String => s
+            case s: String => resolveIncludePath(s)
           }.toSeq
     }
   }
 
+  def resolveIncludePath(path: String) =
+    path.replaceFirst("^~", System.getProperty("user.home").replace("\\", "/"))
+
```

Comment:
On MacOS I was able to successfully use
__includes [ "~/ModelsNetLogo/save-nls/set-parameters.nls" ]
which is what the enhancement request asked for.

I thought that the last replacement would make the following work as well:
__includes [ "~\ModelsNetLogo\save-nls\set-parameters.nls" ]

However it gave the error "Illegal character after backslash

as did
__includes [ "\Users\aab1638\ModelsNetLogo\save-nls\set-parameters.nls" ]

Reply:
The NetLogo/Java format expects the user to use forward slashes, since backslashes are reserved for escaping characters such as double quotes. The backslash replacement code I added was only for the `user.home` property, since on Windows it returns a string with backslashes, but it needs to be forward slashes for NetLogo and Java. I did this instead of restricting the platform, because it seemed like an unnecessary restriction to me, but if you think it shouldn't be allowed on Windows, then I can fix the code to do that instead.